routes/server.go

```diff
@@ -1316,6 +1317,20 @@ func (fes *APIServer) NewRouter() *muxtrace.Router {
 			fes.Stake,
 			PublicAccess,
 		},
+		{
+			"GetStakeForValidatorAndStaker",
+			[]string{"GET"},
+			RoutePathStake + "/{validatorPublicKeyBase58Check:t?BC[1-9A-HJ-NP-Za-km-z]{51,53}}/{stakerPublicKeyBase58Check:t?BC[1-9A-HJ-NP-Za-km-z]{51,53}}",
```

Comment:
Please create a const string for the `t?BC[1-9A-HJ-NP-Za-km-z]{51,53}` regex template so it can be reused